vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileController.kt

```diff
@@ -252,7 +252,7 @@ class RoomProfileController @Inject constructor(
     }
 
     private fun buildEncryptionAction(actionPermissions: RoomProfileViewState.ActionPermissions, roomSummary: RoomSummary) {
-        if (!roomSummary.isEncrypted) {
+        if (!roomSummary.isEncrypted && RoomUtils.getRoomType(roomSummary) != TchapRoomType.FORUM) {
```

Comment:
Please comment the full block `if (!roomSummary.isEncrypted) {...}`
We don't need it in Tchap for the moment